src/resmom/linux/alps.c

```diff
@@ -3050,27 +3051,28 @@ inventory_to_vnodes(basil_response_t *brp)
 	char	*attr;
 	vnl_t	*nv = NULL;
 	int	ret;
+	char	vname[128];
```

Comment:
Should we be using a macro rather than a constant here? If there isn't a macro that defines how long a vnode name may be, then one should be created.